src/Raven.Server/Config/Categories/BackupConfiguration.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.IO;
+using System.Linq;
+using Microsoft.Extensions.Configuration;
+using Raven.Server.Config.Attributes;
+using Raven.Server.Config.Settings;
+using Raven.Server.ServerWide;
+
+namespace Raven.Server.Config.Categories
+{
+    public class BackupConfiguration : ConfigurationCategory
+    {
+        [Description("Local backups are allowed to be created under this root path. If a relative path is specified, it will be located under the application base directory.")]
```

Comment:
What if the local app directly is not part of the local root path?
Instead, we need to say that if a relative path is used, it is placed under the LocalRootPath

Reply:
The description is a mistake, I'll fix it.
The logic behind the path is like this:
If LocalRootPath is specified, the user doesn't have a choice. The backup will be in LocalRootPath\{DatabaseName}\
If LocalRootPath is not specified there is no change in behavior.

Reply:
No, the logic there should be:
If `LocalRootPath` is specified, error if they are trying to set a value that isn't in the `LocalRootPath` (case _sensitive_, because Linux).

Reply:
Reasoning, I might want to set `E:\` as the backup root, and let the user figure it out later.